runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/environment/EmbeddedEnvironmentFactory.java

```diff
@@ -102,6 +102,7 @@ public RemoteEnvironment createEnvironment(Environment environment, String worke
                     options,
                     loggingServer.getApiServiceDescriptor(),
                     controlServer.getApiServiceDescriptor(),
+                    null,
```

Comment:
Setting `null` seems a bit scary. Is there a better way to specify this value? Like a default/empty value instead of setting a `null`?

Reply:
cc @kennknowles any suggestions?